docs/design_decisions.md

```diff
@@ -185,4 +185,4 @@ Er kan gekozen worden de historische voorkomens te raadplegen binnen een periode
 Het antwoordbericht van /verblijfplaatshistorie bevat het property burgerservicenummer van de ingeschreven natuurlijk persoon, plus _links en _embedded met de historische voorkomens van de verblijfplaats. Hierin zitten ook de properties periodeVan en periodeTotEnMet.
 Het antwoordbericht van /verblijfstitelhistorie bevat de historische voorkomens van de property verblijfstitel van de ingeschreven natuurlijk persoon en de properties periodeVan en periodeTotEnMet.
 Het antwoordbericht van /partnerhistorie bevat het property burgerservicenummer van de ingeschreven natuurlijk persoon, plus _links en _embedded met de historische voorkomens van de partner. Hierin zitten ook de properties periodeVan en periodeTotEnMet.
-Het antwoordbericht van /bewoningshistorie bevat de lijst voorkomens met elk de properties periodeVan en periodeTotEnMet, plus _links en _embedded met het betreffende adres en de betreffende persoon (bewoner). De bewoningshistorie kan worden gezocht met dezelfde queryparameters als voor resource bewoning.
+Het antwoordbericht van /bewonershistorie bevat de lijst voorkomens met elk de properties periodeVan en periodeTotEnMet, plus _links en _embedded met het betreffende verblijfsadres met de relatie naar de betreffende bewoners (resource ingeschreven natuurlijke persoon). De bewonershistorie kan worden gezocht met dezelfde queryparameters als voor resource bewoners.
```

Comment:
Dank je Cathy voor je bijdrage.
"lijst voorkomens met elk de properties periodeVan en periodeTotEnMet, plus _links en _embedded met het betreffende verblijfsadres met de relatie naar de betreffende bewoners" is denk ik niet helemaal juist.
moet dit niet zijn:
lijst voorkomens met elk de properties periodeVan en periodeTotEnMet en het betreffende verblijfsadres, plus _links en _embedded met de relatie naar de betreffende bewoners"?
Dan wordt het verblijfsadres niet als relatie (embed) opgenomen, maar op het zelfde niveau als de periode.

Reply:
Klopt!

Reply:
Klopt! Dank!